chia/wallet/cc_wallet/cc_utils.py

```diff
@@ -103,6 +101,7 @@ def spend_bundle_for_spendable_ccs(
     spendable_cc_list: List[SpendableCC],
     inner_solutions: List[Program],
     sigs: Optional[List[G2Element]] = [],
+    extra_deltas: Optional[List[int]] = None,
```

Comment:
```suggestion
    extra_deltas: List[int] = [],
```
Then you don't need to check it against `None` later.

Note that similarly `sigs` could really just be of type `List[G2Element]`